src/main/connector/system/Windows/Windows.yaml

```diff
@@ -149,12 +149,13 @@ monitors:
         serviceInformation:
           type: wmi
           namespace: root\CIMv2
-          query: SELECT Name,ProcessId,State FROM Win32_Service WHERE StartMode != 'Disabled'
+          query: SELECT Name,ProcessId,State,Description FROM Win32_Service WHERE StartMode != 'Disabled'
       mapping:
         source: ${source::serviceInformation}
         attributes:
           id: $1
           processId: $2

```

Comment:
If `processId` is part of the attributes, it will break the timeseries each time the service starts (it will change the value of `processId`). However, we could do something innovative and collect performance metrics (CPU, memory, IO) from this `processId` and attach such performance metrics to the service. Not for now however, future!